### PR TITLE
Add support PyQt5 for OpenBSD 6.x!

### PR DESCRIPTION
**Pull request by hucste**
Add new support for OpenBSD 6.x with PyQt5, as reply for #213!

### CODE REVIEW

File: webview/qt.py [hunk 1]
```diff
@@ -30,7 +31,7 @@
     from PyQt5.QtCore import QT_VERSION_STR
     _qt_version = [int(n) for n in QT_VERSION_STR.split('.')]
 
-    if _qt_version >= [5, 5]:
+    if _qt_version >= [5, 5] and platform.version() == 'Linux':
```
Comment (r0x0r) [inline webview/qt.py]: This should probably be `and platform.version() != 'OpenBSD'`, as QT can be used on OSX as well.

File: webview/qt.py [hunk 2]
```diff
@@ -154,7 +155,7 @@ def __init__(self, uid, title, url, width, height, resizable, fullscreen,
         self.evaluate_js_trigger.connect(self.on_evaluate_js)
         self.set_title_trigger.connect(self.on_set_title)
 
-        if _qt_version >= [5, 5]:
+        if _qt_version >= [5, 5] and platform.version() == 'Linux':
```
Comment (r0x0r) [inline webview/qt.py]: Same here

File: webview/qt.py (hunk 1 above)
Comment (hucste) [reply webview/qt.py]: Ok, i attempt! ;)

Comment (hucste) [reply webview/qt.py]: Not run! :(

Comment (hucste) [reply webview/qt.py]: But this run:
```if _qt_version >= [5, 5] and (platform.version() == 'Linux' or platform.version() == 'Darwin'):```

One idea?!

Comment (hucste) [reply webview/qt.py]: OK, it seems i've a problem with the test: 
```
$ python3 
Python 3.6.2 (default, Oct  2 2017, 09:52:18) 
[GCC 4.2.1 Compatible OpenBSD Clang 4.0.0 (tags/RELEASE_400/final)] on openbsd6
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.version()
'GENERIC.MP#7'
>>> print(f'OS: '+ platform.version())
OS: GENERIC.MP#7
```

```
$ python 
Python 2.7.14 (default, Oct  2 2017, 09:52:26) 
[GCC 4.2.1 Compatible OpenBSD Clang 4.0.0 (tags/RELEASE_400/final)] on openbsd6
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> print 'OS : %s' % platform.version()
OS : GENERIC.MP#7
```

I think that's not normal!

Comment (hucste) [reply webview/qt.py]: Ok, it's `platform.system()` to detect and not `platform.version()`!

Comment (hucste) [reply webview/qt.py]: OK, this corrected with https://github.com/r0x0r/pywebview/pull/215/commits/a780407d84bec60a98acf51b16529cb2b6d6d9e3! 

So good for you?!

Comment (r0x0r) [reply webview/qt.py]: Yes, it is fine now. Thanks!

**Pull request merged**